README.md

```diff
@@ -1 +1,211 @@
-# animated_book_list
\ No newline at end of file
+![banner](https://i.imgur.com/ppKK1L0.jpeg)
+
+# Flutter Animated Book Widget
+
+![build](https://github.com/the-d-velopers/animated_book_list/actions/workflows/code_metrics.yml/badge.svg?branch=master)
```

Comment:
animated_book_list

Reply:
I'll change the name of the repo later

Reply:
Well, that github action info comes from the github repo. Is not from the name of the package.